core/src/main/python/wlsdeploy/tool/discover/domain_info_discoverer.py

```diff
@@ -40,13 +41,18 @@ def discover(self):
         """
         _method_name = 'discover'
         _logger.entering(class_name=_class_name, method_name=_method_name)
+        self.add_admin_credentials()
         model_top_folder_name, result = self.get_domain_libs()
         discoverer.add_to_model_if_not_empty(self._dictionary, model_top_folder_name, result)
         model_top_folder_name, result = self.get_user_env_scripts()
         discoverer.add_to_model_if_not_empty(self._dictionary, model_top_folder_name, result)
         _logger.exiting(class_name=_class_name, method_name=_method_name)
         return self._dictionary
 
+    def add_admin_credentials(self):
+        self._dictionary[model_constants.ADMIN_USERNAME] = alias_constants.PASSWORD_TOKEN
```

Comment:
Can the Admin username be discovered?

Reply:
No. It is a property field that becomes hidden after the writeDomain is performed.